blocks/hero-horizontal-tabs/hero-horizontal-tabs.css

```diff
@@ -70,13 +79,19 @@ main .hero-horiz-tabs-text > p {
   font-size: var(--body-font-size-s);
 }
 
+main div.no-image .hero-horiz-tabs-text > p {
+  color: var(--white);
+  line-height: 1.5em;
+}
+
 main .hero-horiz-tabs-img picture {
   display: block;
   height: 15rem;
 }
 
 main .hero-horizontal-tabs.block.no-image {
-  padding-top: 15rem;
+  padding: 0;
+  margin: auto;
```

Comment:
This margin makes it so this page loses the spacing between the hero and the tabs content
https://130-hero-layout-issue--sunstar--hlxsites.hlx.page/careers/career-opportunities

Reply:
@dnbute  I believe the spacing should come from the next section.